app-tests/next-layout-admin/src/pages/_document.js

```diff
@@ -1,3 +1,5 @@
+import React from 'react';
+
```

Comment:
Please go back through all of these next.js files and remove the React imports that were added due to the lint rule being broken in the override for docs.